### ISSUE TITLE
Bookmarklets does not work/run at the new tab page.

### ISSUE THREAD

**Issue by Githuby3** [brave/brave-browser#31452]
## Description 
Bookmarklets no longer runs on click at the new tab page.

## Steps to Reproduce

   1. Add a bookmarklet to the bookmarkbar
   2. Went to the new tab screen
   3. Click on the bookmarklet in the bookmarkbar
   4. Now open any website
   5. And do step 3. again

## Actual result:
Bookmarklet doesn't run/react on click/interaction with it

## Expected result:
Bookmarket runs/react on click/interaction with it

## Reproduces how often: 
Easily reproduced

## Brave version (brave://version info)
1.52.129 Chromium: 114.0.5735.198 (Offizieller Build) (32-Bit) 

## Version/Channel Information:

- Can you reproduce this issue with the current release? 
Yes.
- Can you reproduce this issue with the beta channel? 
Yes.

## Other Additional Information:

- Does the issue resolve itself when disabling Brave Shields?
No.
- Does the issue resolve itself when disabling Brave Rewards? 
No.

**Comment by rebron** [comment]
Hi @Githuby3 What's the bookmarklet that you're trying to use on the new tab page? It sounds like you're saying the bookmarklet doesn't work when you use the new tab page first but it does work if you don't trigger it on the new tab page?

**Comment by Githuby3** [comment]
No. I mean it doesn't react if you click at the Bookmarklet in the New Tab Page, but if you open a website and click there on the bookmarklet it works, if needed I can make a video of it.

The bookmarklet I used:
javascript:void(() => {prompt('UUID:', self.crypto.randomUUID())})()

I also tried others, same result.

**Comment by Githuby3** [comment]
> No. I mean it doesn't react if you click at the Bookmarklet in the New Tab Page, but if you open a website and click there on the bookmarklet it works, if needed I can make a video of it.
> 
> The bookmarklet I used: javascript:void(() => {prompt('UUID:', self.crypto.randomUUID())})()
> 
> I also tried others, same result.
